nomos-da/kzgrs-backend/src/global.rs

```diff
@@ -1,7 +1,30 @@
-use kzgrs::{global_parameters_from_randomness, GlobalParameters};
+use kzgrs::{global_parameters::global_parameters_from_file, GlobalParameters};
 use once_cell::sync::Lazy;
 
 pub static GLOBAL_PARAMETERS: Lazy<GlobalParameters> = Lazy::new(|| {
-    let mut rng = rand::thread_rng();
-    global_parameters_from_randomness(&mut rng)
+    // TODO: This is for development only, the seed needs to come from the trusted setup
+    // configuration.
+
+    global_parameters_from_file("/Users/bacv/code/logos/nomos-node/tests/kzgrs/kzgrs_test_params")
```

Comment:
Probably we can define a relative path or something for now?
I think probably this needs a bit of a refactor. Probably we should ditch `once_cell` as it is gonna be cumbersome. And will push global params as an actual parameter that we can feed. All to be done in a later PR. For now lets do the path relative otherwise it would just work on your machine :D